test/mocking_classes.py

```diff
@@ -126,6 +126,77 @@ def rand_step(self, tensordict: Optional[TensorDictBase] = None) -> TensorDictBa
         return self.step(tensordict)
 
 
+class MockBatchedLockedEnv(EnvBase):
+    def __init__(self, device, batch_size=None):
+        super(MockBatchedLockedEnv, self).__init__(device=device, batch_size=batch_size)
+        self.action_spec = NdUnboundedContinuousTensorSpec((1,))
+        self.input_spec = CompositeSpec(
+            action=NdUnboundedContinuousTensorSpec((1,)),
+            observation=NdUnboundedContinuousTensorSpec((1,)),
+        )
+        self.observation_spec = CompositeSpec(
+            next_observation=NdUnboundedContinuousTensorSpec((1,))
+        )
+        self.reward_spec = NdUnboundedContinuousTensorSpec((1,))
+        self.counter = 0
+
+    def set_seed(self, seed: int) -> int:
```

Comment:
can we do set_seed = MockSerialEnv.set_seed or something like that, not to re-write entire methods? This does not seem to depend on batch size.